util/test/checkChplInstall

```diff
@@ -177,9 +177,15 @@ if [ ${COMPILE_STATUS} -ne 0 ]; then
     exit 1
 fi
 
+
     # no errors on stdout/stderr?
     # first, apply "prediff"-like filter to remove gmake "clock skew detected" warnings, if any
-TEST_COMP_OUT=$( grep < ${TMP_TEST_DIR}/chpl.out -Ev '^g?make(\[[0-9]+\])?: *[Ww]arning: *(File .* has modification time .* in the future|Clock skew detected\. *Your build may be incomplete\.) *$' 2>&1 )
+TEST_COMP_OUT=$(<${TMP_TEST_DIR}/chpl.out)
+if [ -n "${TEST_COMP_OUT}" ]; then
+    TEST_COMP_OUT=$( grep < "${TMP_TEST_DIR}/chpl.out" -v \
```

Comment:
Should this be grepping `TEST_COMP_OUT` instead of `"${TMP_TEST_DIR}/chpl.out"`?

Reply:
No. Line 183 means it doesn't matter.

Reply:
Doesn't matter as in either would work? Seems like it'd be a little better/cleaner to use `TEST_COMP_OUT`  since that's the "is not empty" condition we're checking